_blog.yml

```
@@ -5961,3 +5961,15 @@
     - community
     - datasets
     - hub
+
+- local: vlms-2025
+  title: "Vision Language Models (Better, faster, stronger)"
```

```suggestion
  title: "Vision Language Models (Better, Faster, Stronger)"
```

for consistency maybe?